### PR TITLE
dotCMS/core#22337 update nx to 13.10.3

### PR DESCRIPTION
**Pull request by oidacra**
Step 1: Update Nx
Step 2: Update other packages

### CODE REVIEW

File: ui/apps/dotcms-block-editor/project.json [hunk 1]
```diff
@@ -39,9 +39,9 @@
                         }
                     ],
                     "optimization": true,
-                    "outputHashing": "all",
+                    "outputHashing": "none",
```
Comment (fmontes) [inline ui/apps/dotcms-block-editor/project.json:42]: We sure? we might want this for cache no?

File: ui/apps/dotcms-ui/src/app/api/services/dot-contentlet-locker/dot-contentlet-locker.service.ts [hunk 2]
```diff
@@ -15,13 +15,13 @@ export class DotContentletLockerService {
      * @returns Observable<any>
      * @memberof PageViewService
      */
-    lock(inode: string): Observable<DotContentletLockResponse> {
+    lock(inode: string): Observable<DotCMSResponse<DotContentletLockResponse>> {
```
Comment (fmontes) [inline ui/apps/dotcms-ui/src/app/api/services/dot-contentlet-locker/dot-contentlet-locker.service.ts:18]: Did you test that this is working? I mean, beyond that TSC compiles?

File: ui/apps/dotcms-ui/src/app/api/services/dot-license/dot-license.service.ts [hunk 3]
```diff
@@ -108,7 +108,7 @@ export class DotLicenseService {
             return url.indexOf(item.url) === 0;
         });
         if (urlMatch.length) {
-            this.unlicenseData.next(...urlMatch);
+            this.unlicenseData.next(urlMatch[0]);
```
Comment (fmontes) [inline ui/apps/dotcms-ui/src/app/api/services/dot-license/dot-license.service.ts:111]: This looks like something that can break functionality. Did you run this?

File: ui/apps/dotcms-ui/src/app/portlets/dot-edit-page/content/services/dot-page-state/dot-page-state.service.ts [hunk 4]
```diff
@@ -228,11 +228,15 @@ export class DotPageStateService {
         return this.loginService.auth.loginAsUser || this.loginService.auth.user;
     }
 
-    private getLockMode(workingInode: string, lock: boolean): Observable<string> {
+    private getLockMode(workingInode: string, lock: boolean): Observable<string | string[]> {
         if (lock === true) {
-            return this.dotContentletLockerService.lock(workingInode).pipe(pluck('message'));
+            return this.dotContentletLockerService
+                .lock(workingInode)
+                .pipe(map((res) => res.messages));
```
Comment (fmontes) [inline ui/apps/dotcms-ui/src/app/portlets/dot-edit-page/content/services/dot-page-state/dot-page-state.service.ts:235]: Why no pluck here?

File: ui/decorate-angular-cli.js [hunk 5]
```diff
@@ -62,7 +62,7 @@ function symlinkNgCLItoNxCLI() {
 
 try {
   symlinkNgCLItoNxCLI();
-  require('@nrwl/cli/lib/decorate-cli').decorateCli();
+  require('nx/src/adapter/decorate-cli').decorateCli();
```
Comment (fmontes) [inline ui/decorate-angular-cli.js:65]: Did you did this manually?

File: ui/libs/dotcms-js/src/lib/core/core-web.service.ts [hunk 6]
```diff
@@ -184,6 +184,7 @@ export class CoreWebService {
                     return new ResponseView<T>(resp);
                 }
             }),
+            tap((resp) => console.log(resp)),
```
Comment (fmontes) [inline ui/libs/dotcms-js/src/lib/core/core-web.service.ts:187]: remove.

File: ui/libs/dotcms-js/src/lib/core/util/response-view.ts [hunk 7]
```diff
@@ -30,6 +33,10 @@ export class ResponseView<T = any> {
         return this.headers.get(headerName);
     }
 
+    get body(): DotCMSResponse<T> {
+        return this.bodyJsonObject;
```
Comment (fmontes) [inline ui/libs/dotcms-js/src/lib/core/util/response-view.ts:37]: Ohhh I see

File: ui/libs/dotcms-js/src/lib/core/util/response-view.ts [hunk 8]
```diff
@@ -13,11 +13,14 @@ import { DotCMSResponse } from '../core-web.service';
...
  * </code>
  */
+
+// TODO: Refactor this to an interceptor
 export class ResponseView<T = any> {
     private bodyJsonObject: DotCMSResponse<T>;
     private headers: HttpHeaders;
 
     public constructor(private resp: HttpResponse<DotCMSResponse<T>>) {
+        console.info('ResponseView class', resp);
```
Comment (fmontes) [inline ui/libs/dotcms-js/src/lib/core/util/response-view.ts:23]: remove.

**Review by fmontes — CHANGES_REQUESTED** [review]

File: ui/libs/dotcms-js/src/lib/core/util/response-view.ts (hunk 8 above)
Comment (oidacra) [reply ui/libs/dotcms-js/src/lib/core/util/response-view.ts:23]: I will add a rule to lint to not allow console.

File: ui/decorate-angular-cli.js (hunk 5 above)
Comment (oidacra) [reply ui/decorate-angular-cli.js:65]: The Nx update.

File: ui/apps/dotcms-block-editor/project.json (hunk 1 above)
Comment (oidacra) [reply ui/apps/dotcms-block-editor/project.json:42]: In dotcms-block-editor we merge all to one file, and we need the same name always `cat dist/apps/dotcms-block-editor/{generator-runtime,runtime,polyfills,main}.js > dotcms-block-editor.js`

File: ui/apps/dotcms-ui/src/app/portlets/dot-edit-page/content/services/dot-page-state/dot-page-state.service.ts (hunk 4 above)
Comment (oidacra) [reply ui/apps/dotcms-ui/src/app/portlets/dot-edit-page/content/services/dot-page-state/dot-page-state.service.ts]: I used pluck again.

File: ui/apps/dotcms-ui/src/app/api/services/dot-contentlet-locker/dot-contentlet-locker.service.ts (hunk 2 above)
Comment (oidacra) [reply ui/apps/dotcms-ui/src/app/api/services/dot-contentlet-locker/dot-contentlet-locker.service.ts]: Yes, its works.

**Pull request closed without merging**